src/cata_tiles.h

```diff
@@ -536,7 +536,7 @@ class cata_tiles
         void draw_cone_aoe_frame();
         void void_cone_aoe();
 
-        void init_draw_bullet( const tripoint &p, std::string name );
+        void init_draw_bullet( const tripoint &p, std::string name, int rotation = 0 );
```

Comment:
No need for default parameter that isn't used anywhere
```suggestion
        void init_draw_bullet( const tripoint &p, std::string name, int rotation );
```